crucible-mir-comp/src/Mir/Compositional/Clobber.hs

```diff
@@ -180,9 +182,10 @@ clobberGlobals sym loc nameStr cs = do
                 Nothing -> error $ "couldn't find static def for " ++ show defId
         let tpr = globalType gv
         let shp = tyToShapeEq (cs ^. collection) (static ^. M.sTy) tpr
+        let nameStr' = nameStr ++ "_" ++ show defId
```

Comment:
I'm not quite sure what this `nameStr'` is supposed to represent. For example, in the call site [here](https://github.com/GaloisInc/saw-script/blob/be40a282ebf1694988a4c51e7c03068a27385766/crucible-mir-comp/src/Mir/Compositional/Override.hs#L345), my understanding is that this would cause things like `run_spec_clobber_globals_foo/123456::bar::baz` to be printed, which (AFAICT) isn't actually the name of an identifier. Would it make more sense to print it as `run_spec_clobber_globals (foo/123456::bar::baz)`?

Reply:
I switched it as you suggested (in a0b106f5d15c07a2b0e006d5fd91e9573f67ad3b), so it now produces output like:

```
clobberImmutSymbolic: Enums not currently supported in overrides, for clobber_globals (test_promoted_enum/b05a7ae7::f_test[0]::ALLOC0[0])
```

The string passed here is also used as the name for symbolic variables generated by `clobber_globals`.  I was initially thinking it would be weird to have spaces in those names, but I guess it's also plenty weird to add `:` `/` `[` `]`, so we might as well include spaces too and make the output clearer.